### ISSUE TITLE
Handling interaction?

### ISSUE THREAD

**Issue by joyrexus** [borismus/webvr-boilerplate#41]
Curious as to how to handle interaction when in VR/cardboard mode.  I'm particularly interested in implementing streetview style expeditions along the lines of [walkinto](http://walkinto.in/).

Any suggestions on how to handle user input to, say, switch scenes sans mouse/touch events?  The [vr.chromeexperiments](https://vr.chromeexperiments.com/) site handles this nicely.

**Comment by cindyloo** [comment]
also curious about this too.  how could/would pointerlock movement controls work in cardboard?

**Comment by brianpeiris** [comment]
I think boris' aim with the polyfill and the boilerplate was simply to
provide head tracking and a basic starter template. So it might not make
sense to add movement controls and interaction to this project. It sounds
like a job for helper libraries instead.
I'm aware of vreticle (https://github.com/neuman/vreticle) for look-based
interaction, for example.
webvr-starter-kit (https://github.com/povdocs/webvr-starter-kit) may also
provide more functionality.
In short, it's up to the community to build missing WebVR libraries.

**Comment by brianpeiris** [comment]
Of course, if it wasn't obvious, there are a ton of existing three.js helpers (e.g. FirstPersonControls) which are not VR-specific but can still be adapted for use in VR.

**Comment by cindyloo** [comment]
right, I am using both pointerlock and first person controls in 'web' mode. I guess it shouldn't be too hard to get a binary move/stop moving switch going in vr mode...

**Comment by borismus** [comment]
Lots of options. Boilerplate doesn't try to be opinionated about it. Take a look at https://play.google.com/store/apps/details?id=com.google.vr.cardboard.apps.designlab&hl=en. Also take a look at the YouTube 360 UI (on Android).

**Comment by joyrexus** [comment]
Appreciate the feedback, esp the vreticle tip, which is exactly what I was looking for to create [fuse targets](https://www.google.com/design/spec-vr/interactive-patterns/controls.html).
